src/core/gui/sidebar/previews/layer/SidebarPreviewLayers.cpp

```diff
@@ -107,12 +107,12 @@ void SidebarPreviewLayers::updatePreviews() {
         return;
     }
 
-    int layerCount = page->getLayerCount();
+    auto layerCount = page->getLayerCount();
 
     size_t index = 0;
-    for (int i = layerCount; i >= 0; i--) {
+    for (auto i = layerCount + 1; i--;) {
```

Comment:
Decreasing loop on unsigned variable. Is there a more readable way?

Reply:
The modern-cpp approach would be a reverse iterator. 
But this might be more readable:?
```suggestion
    for (auto i = layerCount + 1; i != 0;) {
        --i;
```

Reply:
I would make it explicit that it's unsigned.

Secondly, a good way to iterate backwards over an unsigned counter is to "shift" the count by 1:
```c++
for (unsigned i = n + 1; i > 0; i--) {
  (void)foo[i - 1];
}

// or decrement inside loop

for (unsigned i = n + 1; i > 0;) {
  i--;
  (void)foo[i];
}
```